torch/csrc/utils/disable_torch_function.cpp

```diff
@@ -221,7 +221,7 @@ inline bool has_torch_function_attr(PyObject* obj) {
 
 namespace torch {
 auto check_has_torch_function(PyObject* obj, bool ignore_mode) -> bool {
-  if (!ignore_mode && at::impl::PythonTorchFunctionTLS::get_mode())
+  if (!ignore_mode && at::impl::function_mode_enabled())
```

Comment:
nit: this is pre-existing, but function_mode_enabled() isn't descriptive to people who don't know the full name is "torch function mode" -- torch_function_mode_enabled() would be easier to read

Reply:
Agreed--since this is already a refactor, renamed this everywhere